perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -340,21 +352,46 @@ def Run(benchmark_spec):
 
   fio_command = 'sudo %s --output-format=json %s' % (fio.FIO_PATH,
                                                      REMOTE_JOB_FILE_PATH)
+
+  run_reps = FLAGS.run_for_minutes // MINUTES_PER_JOB
+  if FLAGS.run_for_minutes % MINUTES_PER_JOB != 0:
```

Comment:
If users gives a customized job file and also gives FLAGS.run_for_minutes, the logic is broken and leads to unexpected behavior. The job file will be executed until run_reps repetitions. We can either make FLAGS.run_for_minutes more generic and work with customized job file, or we check fio_jobfile flag.